single_tech_samples/storage/sample1_data_lifecycle/README.md

```diff
@@ -0,0 +1,224 @@
+
+Below listed are the steps to deploy this sample :
+
+- Fork and clone this repository. Navigate to  `single_tech_samples/storage/sample1_data_lifecycle/terraform`.
+
+- The sample depends on the variables to be set before the deployment script is run; You can set these variables by editing the variables.tf and terraform.auto.tfvars files, otherwise the default values provided would be considered.
+
+The script is configuration driven and based on a variable of type map(map(map))).
+
+- First level map represents storage accounts.
+- Second level map represents the containers inside the respective storage account.
+- Third level map represents the data life cycle rules under respective container.
+
+Sample variable format is shown below:
+
+```json
+storage_account_container_config = {
+    "StorageAccountName1" = {
+        "Container1"       = { "LifeCycleAction1" : "NumberOfDays", "LifeCycleAction2" : "NumberOfDays"}
+    }
+    "StorageAccountName2" = {
+        "Container2"       = { "LifeCycleAction1" : "NumberOfDays", "LifeCycleAction1" : "NumberOfDays" }
+    }
+    "StorageAccountName3" = {
+        "Container3"       = { "LifeCycleAction1" : "NumberOfDays" }
+        "Container4"       = {   }
+    }
+}
+```
+
+Configure as many storage accounts, containers and lifecycle rules as needed and run the IAC script provided in this git repo.
+
+Run below commands in the sequence:
+
+```bash
+terraform init
+terraform plan
+terraform apply
+```
+
+   > Note: The script would expect that you are logged in to the Azure account using az login command, before you deploy resources to Azure.
+
+### 2.3. Deployed Resources
+
+The following resources will be deployed as a part of this sample once the script is executed:
+
+- Azure Storage Accounts.
+- Azure Storage Containers.
+- Azure data lifecycle rules.
+
+### 2.4. Deployment validation and Execution
+
+The following steps can be performed to validate the correct deployment and execution of the sample:
+
+Users with appropriate access rights should be able to:
+
+- View the storage accounts in the Azure portal.
+- View the storage containers under the storage accounts through the Azure portal.
+- View the data lifecycle rules created in the Azure portal under storage accounts.
+
+Here are the detail steps on how to execute the sample:
+
+- Checkout the code.
+- Configure the variables as needed under the files variables.tf and terraform.auto.tfvars
+- Here are the sequence of commands to run from the main directory :
+
+   ```bash
+   terraform init
+   terraform plan
+   terraform apply
+   ```
+
+After you run the above commands successfully, you should be able to see the following items under your Azure Subscription:
+
+- Azure Storage Accounts. Sample view is shown below.
+
+![alt text](../common_assets/images/storage_accounts.png "Logo Title Text 1")
+You can observe the appended random strings to make the storage accounts’ names unique since we have used the azurecaf feature.
```

Comment:
Remove 1st person usage "we have used the azurecaf..."

Reply:
Done